optimuspy.py

```diff
@@ -134,8 +134,17 @@ def main(instance_name: str, view_name: str, process_name: str, executions: int,
     with TM1Service(**tm1_args) as tm1:
         original_performance_monitor_state = retrieve_performance_monitor_state(tm1)
         activate_performance_monitor(tm1)
+        all_tm1_cubes = filter(lambda c: not c.startswith("}"), tm1.cubes.get_all_names())
+
+        if cube_name is not None:
+            if cube_name in all_tm1_cubes:
```

Comment:
I think it would be nice if this check was case and space insensitive since object names in TM1 are case and space insensitive.

You could write : `if cube_name in CaseAndSpaceInsensitiveSet(all_tm1_cubes)`

Reply:
So if user passed `General ledger` instead `GeneralLedger` it would still work

Reply:
great suggestion, I had an error pop up with "Pnl" rather than "PnL", this should solve it.